src/io/text_parser.rs

```diff
@@ -50,10 +50,11 @@ const CONTRADICT_CHAR: char = '❌';
 
 pub fn render(
     filename: &str,
-    graph: &Graph,
+    graphs: Vec<Graph>,
     key: &IndexMap<char, u16>,
     width: usize,
 ) {
+    let graph = graphs.last().unwrap(); // text parser does do support progress renders yet
```

Comment:
I don't think text parser will ever have progress renders with blending but we could at least show each vertex which is fully observed in order, or something like that. Anyway for later.